### DISCUSSION TITLE
How do you create Menus and Sub-Menus

### DISCUSSION THREAD

**Discussion by umariani** [vlang/v#13163] [General]
How do you create Menus and Sub-Menus
 - Menu Layout
 - Menu Items
 - Sub-Menus
 - Handle menu events on click

**Comment by JalonSolov** [comment]
This is a discussion for https://github.com/vlang/ui

**Reply by Wajinn** [reply]
> This is a discussion for https://github.com/vlang/ui

Unfortunately, V UI doesn't have their public discussions open and appear to be only on discord.  Maybe something for the V and V UI teams to work out?  When people come from the vlang.io page, it gives the impression that the V UI project is strongly connected and a part of V.  Consequently, it would probably occur to a lot of people to ask questions here.  Maybe its not so bad, as advanced V UI users or contributors might decide to answer.

**Comment by ghost** [comment]
You can't create sub-menu.
`Menu` is barely maintained, so you can't do much with it.
This is true for V-UI in general, but you need to implement the features you want by yourself.
Even some of the basic functions are still undeveloped...
